getting-started/kubernetes/vpp/vm.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Test VPP on a VM-based cluster
+description: Test the VPP dataplane on a simple VM-based cluster
+canonical_url: '/getting-started/kubernetes/vpp/vm'
+---
+
+### VM-based test cluster
+
+In order to test Calico-VPP, you can simply use the test infrastructure to deploy it on a 3-VM cluster. Requirements are :
```

Comment:
To test the VPP dataplane, you can use the test infrastructure to deploy it on a 3-VM cluster. 

**Required**
- An Ubuntu 18.04 or 20.04 machine with:
  - 8 CPU
  - 16 GB RAM
  - 100GB disk
  - {% include open-new-window.html text='kubectl installed' url='https://kubernetes.io/docs/tasks/tools/install-kubectl/' %} : 
  - {% include open-new-window.html text='docker installed' url='https://docs.docker.com/engine/install/ubuntu/' %}: